sagemaker-pipelines/step-decorator/local-mode-example/using_step_decorator_with_local_mode.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        predictions = model.predict(xgboost.DMatrix(x_test))
</pre><p> </p><p>If we change to use XGBRegressor, this would need an update as no need to convert the numpy to DMatrix</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>